Packs/Netmiko/Integrations/Netmiko/Netmiko_description.md

```diff
@@ -1,17 +1,17 @@
-
 ## Netmiko SSH module integration
 
-This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, visit the following URL:
+This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, please visit the below URL:
 
 [Netmiko Platforms.md on Github](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md)
 
 ## Configuration Parameters
 
- - *Name* - Integration instance name.
- - *Platform* - The Netmiko-specific platform name.
- - *Hostname* - The IP address, hostname, or FQDN to connect to over SSH.
- - *Port* - The port to use for the SSH connection.
- - *Credentials* - The credentials should be the same as the Tanium client.
+ - ***Name*** - Integration instance name
+ - ***Platform*** - The Netmiko-specific platform name
+ - ***Hostname*** - The IP address, Hostname, or FQDN to connect to over SSH
+ - ***Port*** - The port to use for the SSH connection
+ - ***Credentials*** - The credentials should be the same as the Tanium client.
+ - ***TimeoutOverride*** - Override the default read timeout value used for a SSH connection (useful for slow-responding devices).
```

Comment:
```suggestion
 - ***Override the default timeout value*** - Override the default read timeout value used for a SSH connection (useful for slow-responding devices).
```
The parameters here should have the value of the display value of the parameter.